plugins/catalog-react/report-alpha.api.md

```diff
@@ -343,6 +343,7 @@ export const EntityContextMenuItemBlueprint: ExtensionBlueprint<{
 export type EntityContextMenuItemParams = {
   useProps: UseProps;
   icon: JSX_2.Element;
+  filter?: (entity: Entity) => boolean;
```

Comment:
Hmm, I'm a little torn on this.

I wonder if we should really be doubling down on the text query format everywhere and align it with cards like we do [here](https://github.com/backstage/backstage/pull/21480).

It makes it easier to configure this in config I guess rather than having to create your own extension override in code and put it in a module.

I'll defer to @camilaibs and @Rugvip though.

Reply:
@benjdlambert The reason I didn't do it that way is because `buildFilterFn` which is used to create the filter function from the filter expression/function lives in the `catalog` plugin itself. Though I suppose it wouldn't be a huge lift to refactor it out into `catalog-react`. If that sounds reasonable I can go ahead and do that.

Reply:
Right, so when you arrive at the component props it's [already been converted](https://github.com/backstage/backstage/blob/e1a471f81bddb6fe3dca8a0be8e43d94cf334484/plugins/catalog/src/alpha/entityContents.tsx#L91) into purely a callback, up in the blueprint loader. 🤔